Sources/SwiftUITooltip/TooltipModifier.swift

```diff
@@ -201,7 +202,7 @@ struct Tooltip_Previews: PreviewProvider {
         
         
         return VStack {
-            Text("Say...").tooltip(config: config) {
+            Text("Say...").tooltip(true, config: config) {
```

Comment:
The `enabled` argument should not be required here, so I'd remove the `true` literal since it's not doing anything special here